### DISCUSSION TITLE
Dataset?

### DISCUSSION THREAD

**Discussion by johohoro** [SDGforskning/SDGstrings_wos#62] [General]
Impressive work. I'm just wondering if a dataset, structured as a table, containing terms and corresponding sdgs would be useful to a broader public, as well as documentation on how the terms ought to be combined. The python code could in turn read the dataset. Splitting code and terms would make it easier to add and edit terms without affecting code or vice versa. I would for example be interested in using a term-database and implementing them over a set of records using R.

**Comment by Caroline-A** [comment]
Hi @johohoro - I realise that this post is very old, I did not get a notification and have not checked the Discussions - I apologise! This is super interesting, if you are still interested you are welcome to get in touch or perhaps give an example. We have a goal to improve the python script and would love input on what would be useful.
